packages/browser-sdk/src/context.ts

```diff
@@ -1,15 +1,13 @@
 export interface CompanyContext {
-  id: string | number;
-  [key: string]: string | number;
+  [key: string]: string | number | undefined;
 }
 
 export interface UserContext {
-  id: string | number;
-  [key: string]: string | number;
+  [key: string]: string | number | undefined;
```

Comment:
could you not mark the `id` here as optional. Same goes to name and email to be explicitly listed but optional.